### ISSUE TITLE
Print out learning_rate in the progress bar

### ISSUE THREAD

**Issue by benihime91** [PyTorchLightning/pytorch-lightning#3191]
Hello!
I am very new to `pytorch_lightning`.
I am using a `lr_scheduler` which changes the `learning_rate` each step. I would like to print out the value of the `lr` at each step. Is there any way to print out ` learning rate` in the progress bar alongside the `loss` & `metrics`
